### ISSUE TITLE
Wrong position of resulting bounds

### ISSUE THREAD

**Issue by tim-kilian** [bpmn-io/diagram-js-direct-editing#6]
I've got a problem, that the bounds returned in the update function are interchanged. `x` should be `bounds.left` and `y` should be `bounds.top`. Next the method getBoundingClientRect is returning the absolute value from the top of the screen. When there is something above or left of the SVG-DOM its not correct. 

- [x] swaping `x` and `y`
- [x] correct bounds

```
  var bounds = this.$textbox.getBoundingClientRect();

  if (text !== active.context.text || this.resizable) {
    active.provider.update(active.element, text, active.context.text, {
      x: bounds.top,
      y: bounds.left,
      width: bounds.width,
      height: bounds.height
    });
  }
```

**Comment by nikku** [comment]
Is this a code snipped you came up with or a snippet existing in our code base? If the later is the case, could you link to the relevant source file you found it in?

Alternatively, if you already found this to be a bug, could you file a PR that fixes it (and [test cases](https://github.com/bpmn-io/diagram-js-direct-editing/blob/master/test/DirectEditingSpec.js) that assert the correct behavior).

**Comment by tim-kilian** [comment]
Indeed it is. Sry, I didn't know that I could create a permalink. I'll create a PR soon.
 
https://github.com/bpmn-io/diagram-js-direct-editing/blob/1bdfc2ffd182306511c7f88bb3fc53ee9632cdc0/lib/DirectEditing.js#L92-L101

**Comment by nikku** [comment]
Looking forward for your PR.

**Comment by nikku** [comment]
Closed via #8
